server/usr/share/pulp/templates/directory_index.html

```diff
@@ -0,0 +1,31 @@
+
```

Comment:
There isn't enough JavaScript in here. Please add minified JavaScript that nobody can tell what it does.